### PR TITLE
Include all users with commit access in the submit queue whitelist

### PR DESCRIPTION
**Pull request by eparis**
Continue to include those in the whitelist.txt

### CODE REVIEW

**Comment by eparis** [comment]
@mikedanese @lavalamp @brendandburns You three have played in the submit queue

File: submit-queue/whitelist.txt [hunk 1]
```diff
@@ -1,41 +1,6 @@
-brendandburns
-thockin
-mikedanese
-a-robinson
-saad-ali
-lavalamp
```
Comment (lavalamp) [inline submit-queue/whitelist.txt]: I think your script is trigger happy...

Comment (eparis) [inline submit-queue/whitelist.txt]: you have commit access, so you don't need to be in the whitelist.

Comment (eparis) [inline submit-queue/whitelist.txt]: better explanation, the point of the PR was to pull people with commit access from github and consider them 'whitelisted'  I'm not trying to maintain that list in the whitelist.txt.  We just pull it when the submit-queue launches. So now we only have to maintain in this file those people without commit access that we want to be able to use the submit queue without the extra label.

Comment (lavalamp) [inline submit-queue/whitelist.txt]: Oh, I see, sorry!

File: submit-queue/whitelist.txt [hunk 2]
```diff
@@ -1,41 +1,6 @@
...
-markturansky
-pmorie
-yujuhong
-roberthbailey
-vishh
-deads2k
-bprashanth
-cjcullen
-liggitt
 bgrant0607
```
Comment (lavalamp) [inline submit-queue/whitelist.txt]: bgrant0607 doesn't have committ access?

**Comment by lavalamp** [comment]
LGTM

**Comment by lavalamp** [comment]
I think we shouldn't have enough committers that this would make us run out of quota.

File: submit-queue/whitelist.txt (hunk 2 above)
Comment (eparis) [inline submit-queue/whitelist.txt]: Surprised me so I had to look it up. https://github.com/orgs/kubernetes/teams/kubernetes-maintainers  only bgrant0607-nocc has commit access.

**Comment by eparis** [comment]
Should be about 3ish github API calls each time you start the submit-queue daemon. Not even 3 per poll period.

File: submit-queue/whitelist.txt (hunk 2 above)
Comment (bgrant0607) [inline submit-queue/whitelist.txt]: I could fix that.

Comment (eparis) [inline submit-queue/whitelist.txt]: doesn't matter from my PoV.

**Comment by lavalamp** [comment]
Planning on merging this and rebuilding/starting the merge queue in the coming hour or two.

**Comment by lavalamp** [comment]
Merge queue has been restarted.

**Comment by lavalamp** [comment]
When DNS propagates, the submit queue status should be at http://submit-queue.k8s.io/

Until then, it's at http://104.154.50.45/.

Clearly it could stand to be improved.

**Comment by lavalamp** [comment]
Printing this error now. @brendandburns perhaps the token that you gave it is overly limited? Or maybe that's by design.

```
E0905 00:17:00.926084       1 github.go:263] GET https://api.github.com/orgs/kubernetes/teams?page=1&per_page=100: 403 Must have admin rights to Repository. []
```

**Pull request merged**